profiles/utils.py

```diff
@@ -29,3 +29,15 @@ def requester_can_view_verified_personal_information(request):
         or request.user_auth.data.get("amr")
         in settings.VERIFIED_PERSONAL_INFORMATION_ACCESS_AMR_LIST
     )
+
+
+def requester_has_sufficient_loa_to_perform_gdpr_request(request):
```

Comment:
I think we should either wait for UI to catch up with this feature, or add with a feature flag before merging.

Reply:
Roger, will wait for UI